### DISCUSSION TITLE
Can't We Record Session Logs in Plain Text?

### DISCUSSION THREAD

**Discussion by computerphilosopher** [gravitational/teleport#30891] [Q&A] [answered]
Due to our team's policy, we need to log the account that executed the command, the content of the command, and the time the command was executed in a text format. Is there a way to store session recordings as text rather than in video format?

**Comment by webvictim** [comment, marked as answer]
This is not currently possible.

The easiest way to achieve this if you have or can use newish kernels is to use BPF session recording: https://goteleport.com/docs/server-access/guides/bpf-session-recording/

Your request essentially wants https://github.com/gravitational/teleport/issues/11694 to be implemented, but unfortunately it's not likely to ever happen as it's too easy to obscure commands with obfuscation.
